src/technologies/f.json

```diff
@@ -1704,7 +1707,8 @@
     ],
     "saas": true,
     "scriptSrc": "framerusercontent\\.com",
-    "website": "https://framer.com/sites"
+    "generator": "^Framer",
```

Comment:
Is this supports to be in a `meta` property?

Also can you sort the properties?

Reply:
Yep, was meant to be in a `meta` prop. fixed.

Sorting: done!